aten/src/ATen/native/transformers/attention.cpp

```diff
@@ -653,6 +662,9 @@ std::tuple<Tensor, Tensor> _scaled_dot_product_attention_math(
         // Otherwise, attn_mask represents an additive attention tensor
     }
     auto attn = at::matmul(query, key.transpose(-2, -1)*scaling_factor);
+    if (is_negative_scaling) {
+      attn = -attn;
+    }
```

Comment:
I guess you can make the input of `at::matmul` negative to avoid an extra post-op here?

Reply:
but SymFloat doesn't have neg method: https://github.com/pytorch/pytorch/blob/main/c10/core/SymFloat.h/